release-notes/General/General_Feature_Release_10.3/General_Feature_Release_10.3.1.md

```diff
@@ -43,6 +43,294 @@ Alternatively, you can also pass a button style directly to the `AppendButton` m
 
 ## Other features
 
+#### New table matrix [ID_34645] [ID_34661] [ID_34839] [ID_34879] [ID_34933]
+
+<!-- MR 10.4.0 - FR 10.3.1 -->
+
+It is now possible to create a matrix control in a protocol in a more intuitive way, based on two tables.
+
+For this purpose, three parameters must be configured in the protocol:
+
+- A table parameter for the inputs of the matrix.
+- A table parameter for the outputs of the matrix.
+- A dummy parameter, which contains the matrix mappings and determines where the matrix control is displayed.
+
+> [!NOTE]
+> If a matrix control is configured like this, the severity colors of the crosspoints depend on the alarm monitoring configured for the inputs and outputs tables in the alarm template. This is not configured with the matrix alarm level editor like for classic matrix controls. The crosspoint severity color will be the highest severity color for all monitored column parameters of the outputs table for the relevant row.
```

Comment:
It's only on the outputs table that monitored columns will be reflected onto the crosspoints not on the inputstable

Reply:
I've adjusted this. Could you take another look?